packages/core/src/App/Containers/Layout/app-contents.jsx

```diff
@@ -25,6 +26,9 @@ const AppContents = ({
     pageView,
     pushDataLayer,
 }) => {
+    const [external_link_statue, setExternalLinkStatus] = React.useState(false);
+    const [dialog_status, setDialogStatus] = React.useState(false);
+    const [external_link, setExternalLink] = React.useState(false);
```

Comment:
the default value for `useState` has another type with the expected type for `external_link`